src/Breakpoints.ts

```diff
@@ -4,17 +4,20 @@ import { createRuleSet, createClassName } from "./Utils"
 /**
  * A union of possible breakpoint props.
  */
-export type MediaBreakpointKey = keyof MediaBreakpointProps
+export type BreakpointConstraintKey = keyof MediaBreakpointProps
 
-type ValueBreakpointPropsTuple<T, B> = [T, MediaBreakpointProps<B>]
+type ValueBreakpointPropsTuple<SizeValue, BreakpointKey> = [
+  SizeValue,
+  MediaBreakpointProps<BreakpointKey>
+]
 
 type Tuple = [string, string]
 
 function breakpointKey(breakpoint: string | Tuple) {
   return Array.isArray(breakpoint) ? breakpoint.join("-") : breakpoint
 }
 
-export enum BreakpointKey {
+export enum BreakpointConstraint {
```

Comment:
While this is _technically_ a breaking change for this module's api, it should only affect the internal API. 

The external API is preserved via [this change](https://github.com/artsy/fresnel/pull/155/files#r487422420).